controllers/userController.js

```diff
@@ -0,0 +1,21 @@
+const User = require('../models/userModel');
+
+exports.getAllUsers = async (req, res) => {
+  const users = await User.find();
+
+  res.status(200).json({
+    status: 'success',
+    data: { users }
+  });
+};
+
+exports.createUser = async (req, res) => {
+  const newUser = await User.create(req.body);
+
+  res.status(201).json({
```

Comment:
201 shouldnt return any body, instead it should add location header - where the new resource can be found. For example 

res.set('location', '/api/v1/users/${newUser.id}/`)